tfhe/examples/utilities/params_to_file.rs

```diff
@@ -0,0 +1,172 @@
+use std::fs::{File, OpenOptions};
+use std::io::Write;
+use std::path::Path;
+use tfhe::boolean::parameters::{BooleanParameters, VEC_BOOLEAN_PARAM};
+use tfhe::core_crypto::commons::dispersion::StandardDev;
+use tfhe::core_crypto::commons::parameters::{GlweDimension, LweDimension, PolynomialSize};
+use tfhe::keycache::NamedParam;
+use tfhe::shortint::parameters::multi_bit::ALL_MULTI_BIT_PARAMETER_VEC;
+use tfhe::shortint::parameters::{ShortintParameterSet, ALL_PARAMETER_VEC};
+
+pub trait Spam {
+    fn lwe_dimension(&self) -> LweDimension;
+    fn glwe_dimension(&self) -> GlweDimension;
+    fn lwe_modular_std_dev(&self) -> StandardDev;
+    fn glwe_modular_std_dev(&self) -> StandardDev;
+    fn polynomial_size(&self) -> PolynomialSize;
+}
+
+impl Spam for BooleanParameters {
+    fn lwe_dimension(&self) -> LweDimension {
+        self.lwe_dimension
+    }
+
+    fn glwe_dimension(&self) -> GlweDimension {
+        self.glwe_dimension
+    }
+
+    fn lwe_modular_std_dev(&self) -> StandardDev {
+        self.lwe_modular_std_dev
+    }
+    fn glwe_modular_std_dev(&self) -> StandardDev {
+        self.glwe_modular_std_dev
+    }
+
+    fn polynomial_size(&self) -> PolynomialSize {
+        self.polynomial_size
+    }
+}
+
+impl Spam for ShortintParameterSet {
+    fn lwe_dimension(&self) -> LweDimension {
+        self.lwe_dimension()
+    }
+
+    fn glwe_dimension(&self) -> GlweDimension {
+        self.glwe_dimension()
+    }
+
+    fn lwe_modular_std_dev(&self) -> StandardDev {
+        self.lwe_modular_std_dev()
+    }
+    fn glwe_modular_std_dev(&self) -> StandardDev {
+        self.glwe_modular_std_dev()
+    }
+
+    fn polynomial_size(&self) -> PolynomialSize {
+        self.polynomial_size()
+    }
+}
+
+///Function to print in the lattice_estimator format the parameters
+/// Format:   LWE.Parameters(n=722, q=2^32, Xs=ND.UniformMod(2),
+/// Xe=ND.DiscreteGaussian(56139.60810663548), tag='test_lattice_estimator')
+pub fn format_lwe_parameters_to_lattice_estimator<T: Spam + NamedParam>(
+    param: &T,
+    log_ciphertext_modulus: usize,
+) -> String {
+    let modular_std_dev = log_ciphertext_modulus as f64 + param.lwe_modular_std_dev().0.log2();
+
+    format!(
+        "{}_LWE = LWE.Parameters(\n n = {},\n q ={},\n Xs=ND.UniformMod(2), \n Xe=ND.DiscreteGaussian({}),\n tag='{}_lwe' \n)\n\n",
+        param.name(), param.lwe_dimension().0, (1u128<<log_ciphertext_modulus as u128), 2.0_f64.powf(modular_std_dev), param.name())
+}
+
+///Function to print in the lattice_estimator format the parameters
+/// Format: LWE.Parameters(n=722, q=2^32, Xs=ND.UniformMod(2),
+/// Xe=ND.DiscreteGaussian(56139.60810663548), tag='test_lattice_estimator')
+pub fn format_glwe_parameters_to_lattice_estimator<T: Spam + NamedParam>(
+    param: &T,
+    log_ciphertext_modulus: usize,
+) -> String {
+    let modular_std_dev = log_ciphertext_modulus as f64 + param.glwe_modular_std_dev().0.log2();
```

Comment:
Also, I think the log_ciphertext_modulus should come from the parameter themselves (via the `Spam` trait)
since for example in shortint, the ciphertext modulus comes from the parameters


so in the Spam trait I would add 

```rust
    // For boolean parameters
    fn log_ciphertext_modulus (&self) -> usize{
        32
    }

    // For shortint parameters
    fn log_ciphertext_modulus (&self) -> usize{
       assert!(self.ciphertext_modulus.is_native_modulus));
        64
    }
```